packages/xrpl-crypto/src/ripemd160/browser.ts

```diff
@@ -0,0 +1,6 @@
+import { ripemd160 as nobleImpl } from '@noble/hashes/ripemd160'
+
+import wrapNoble from '../internal/wrapNoble'
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I'd recommend just turning this linting rule off if most of the things in this package don't prefer it.